mailgun/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class HttpResponseException(RuntimeError):
+    def __init__(self, response):
+        self.response = response
```

Comment:
Would we like to have `__str__()` and `__unicode__()` methods for this custom object?